packages/schematics/src/collection/ng-add/index.ts

```diff
@@ -84,7 +82,7 @@ function updatePackageJson() {
       packageJson.dependencies['@ngrx/store'] = ngrxVersion;
     }
     if (!packageJson.dependencies['@ngrx/router-store']) {
-      packageJson.dependencies['@ngrx/router-store'] = routerStoreVersion;
```

Comment:
It's all part of the same set, we should use the same version (they had the same value anyways)